text/phonemes/1-model.jl

```diff
@@ -79,7 +79,7 @@ function train(; kws...)
     @info("Constructing Model...")
     state, encode = Construct_model(args)
 
-    loss(x, yo, y) = sum(crossentropy.(model(x, yo, state, encode), y))
+    loss(x, yo, y) = sum(logitcrossentropy.(model(x, yo, state, encode), y))
```

Comment:
also here, do we want to broadcast?

Reply:
This also had the same problem as the above.